2-environments/modules/env_baseline/variables.tf

```diff
@@ -81,3 +81,9 @@ variable "assured_workload_configuration" {
   })
   default = {}
 }
+
+variable "auto_create_network" {
+  description = "Control whether the 'default' network will be created in the project."
+  type        = bool
+  default     = true
+}
```

Comment:
this should not be a variable, it is a hard coded value when using the project factory method